vc-generator/generators.js

```diff
@@ -1,120 +1,152 @@
 /*!
- * Copyright 2023 Digital Bazaar, Inc. All Rights Reserved
+ * Copyright 2023-24 Digital Bazaar, Inc. All Rights Reserved
```

Comment:
```suggestion
 * Copyright 2023-2024 Digital Bazaar, Inc.
```

Sorry...these keep sneaking in... We also don't need `All Rights Reserved` anywhere. No need to fix these things "everywhere" (now), but we can fix this one at least.